backend/services/main-service/utils/packed_data/boards.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "boardId": 1,
+    "name": "Travel Experiences",
+    "dateOfCreation": "2024-01-01T12:00:00Z",
+    "description": "A board dedicated to sharing travel experiences from around the world.",
+    "username": "user123",
+    "posts": ["Exploring the Alps", "Beach Adventures"],
```

Comment:
posts array contains postIds, which are numbers, but in string format, "1", "3", "5".
This needs to be fixed please.